mobile/native/src/api_calculations/mod.rs

```diff
@@ -0,0 +1,73 @@
+use crate::api_model::Position;
+use bdk::bitcoin;
+use bdk::bitcoin::Denomination;
+use rust_decimal::prelude::ToPrimitive;
+use rust_decimal::Decimal;
+
+/// Calculate the margin in BTC.
+pub fn calculate_margin(opening_price: f64, quantity: f64, leverage: f64) -> u64 {
```

Comment:
Tests would be nice :)

Reply:
yeah, we had them in ItchySats but never pulled them into the PoC (same here...).
I'll add a ticket to re-add them!